screen-reader/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "@capacitor/screen-reader",
+  "version": "0.0.1",
+  "description": "Screen Reader APIs",
+  "main": "dist/plugin.js",
+  "module": "dist/esm/index.js",
+  "types": "dist/esm/index.d.ts",
+  "scripts": {
+    "lint": "npm run prettier -- --check && npm run swiftlint -- lint",
+    "fmt": "npm run prettier -- --write && npm run swiftlint -- autocorrect",
```

```suggestion
    "fmt": "npm run prettier -- --write && npm run swiftlint -- autocorrect  --format",
```

Add the `--format` param